Dockerfile

```diff
@@ -17,7 +17,7 @@ ENV DEFAULT_TERRAFORM_VERSION=1.3.3
 
 # In the official Atlantis image we only have the latest of each Terraform version.
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
-RUN AVAILABLE_TERRAFORM_VERSIONS="0.11.15 0.12.31 0.13.7 0.14.11 0.15.5 1.0.11 1.1.9 1.2.9 ${DEFAULT_TERRAFORM_VERSION}" && \
```

Comment:
This is breaking change that we should call out separately.

Reply:
so we agreed to support only version 1.X to reduce the image size and document the change as a breaking change and announce it in Slack to the community